src/app/spaces/space.service.spec.ts

```diff
@@ -44,7 +44,13 @@ describe('Service: SpaceService', () => {
         {
           provide: WIT_API_URL,
           useValue: "http://example.com"
-        }
+        },
+        {
+          provide: AUTH_API_URL,
+          useValue: 'http://example.com/auth'
+        },
+        UserService,
+        Broadcaster
```

Comment:
The Jasmine tests failed compilation until I added those. I only needed UserService though as a transitive dep of SpaceService.